### PR TITLE
Mark ComponentSLIs as GA in 1.32

### PR DESCRIPTION
**Pull request by tengqm**
Upstream PR: https://github.com/kubernetes/kubernetes/pull/128317
Status: merged

### Issue

Closes: #

### CODE REVIEW

**Comment by Shubham82** [comment]
@tengqm, changes seem good, curious to know, why it is showing `(enabled by default: true)` in the feature state, is it due to the v1.32 branch?

Here is the preview screenshot:

Screenshot

File: content/en/docs/reference/instrumentation/slis.md
Comment (sftim) [inline content/en/docs/reference/instrumentation/slis.md:1]: > With SLI metrics enabled, each Kubernetes component exposes two metrics,
> labeled per healthcheck:

Does this text also need to change? I think the endpoint is now served unconditionally.

File: content/en/docs/reference/command-line-tools-reference/feature-gates/component-slis.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.27"
+    toVersion: "1.31"
+  - stage: stable
+    defaultValue: true
+    fromVersion: "1.32"
+
 ---
 Enable the `/metrics/slis` endpoint on Kubernetes components like
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/component-slis.md:22]: Optionally:
```suggestion
Enable the `/metrics/slis` HTTP endpoint on Kubernetes components like
```

**Review by sftim — COMMENTED** [review]
/lgtm
(for docs)

This change will make the docs less wrong.

**Pull request closed without merging**